qek/data/graphs.py

```diff
@@ -197,9 +197,24 @@ def compute_register(self) -> pl.Register:
         assert pos is not None
         return pl.Register.from_coordinates(coords=pos)
 
-    def compute_sequence(self) -> pl.Sequence:
+    # Default values for the sequence.
+    #
+    # See the companion paper for an explanation.
+    SEQUENCE_DEFAULT_AMPLITUDE_RAD_PER_US = 1.0 * 2 * np.pi
+    SEQUENCE_DEFAULT_DURATION_NS = 660
```

Comment:
Would it make sense to add all these constants to a `constants` or `utils` module ?

Reply:
Good question. Since they're really specific to encoding graphs, I think that I would prefer keeping them here.

Do you see any benefit to putting them in a `constants` module?

FWIW, I wouldn't encourage a `utils` modules, they always end up cluttered with too much stuff that we can never move away afterwards.